.github/workflows/integration.yml

```diff
@@ -22,20 +22,21 @@ on:
 jobs:
 
   build:
-    name: Build and Test
+    name: Build and Test - Redis Stack ${{matrix.redis-stack-version}}
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        redis-stack-version: ['6.2.6-v9', '7.2.0-RC3', 'edge']
```

Comment:
Awesome. But, while here let's update the versions to the latest of each family. Current is 7.2.0-v8 (which is also the latest tag) and 6.2.6-v12.

Reply:
Updated.